internal/runner/service.go

```diff
@@ -123,19 +124,23 @@ func (r *runnerService) findSession(id string) *Session {
 }
 
 func (r *runnerService) Execute(srv runnerv1.RunnerService_ExecuteServer) error {
-	r.logger.Info("running Execute in runnerService")
+	logger := r.logger.With(zap.String("_id", xid.New().String()))
```

Comment:
Why include an `_id` field here? The field will not propagate through the runner so it will only be for top-level log calls in this function. Is there any reason we want those to be distinguished?

Reply:
Exactly, it makes it easier to group logs per single execution.